src/makeCalculator.js

```diff
@@ -36,8 +36,39 @@
  *
  * @return {object}
  */
-function makeCalculator() {
-  // write code here
+function makeCalculator(callback, value) {
+  const resObj = {
```

Comment:
[NAMING] - Use proper names for variables. If function was called `makeFruit`, it should return `fruit`. (from checklist)
You can name your object, for example, 'calculator.'